src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -140,13 +142,15 @@
     <Compile Remove="libraries\System.Text.Json\Serializer\WritePreservedReferences.cs" />
     <Compile Remove="libraries\System.Net.Security\SslStreamTests.Context.cs" />
     <Compile Remove="libraries\System.Formats.Cbor\*.cs" />
+    <Compile Remove="libraries\System.Formats.Tar\*.cs" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' &lt; '6.0')">
     <Compile Remove="libraries\System.Collections\PriorityQueue\Perf_PriorityQueue.cs" />
     <Compile Remove="libraries\Common\AlignedMemory.cs" />
     <Compile Remove="libraries\System.IO.FileSystem\Perf.RandomAccess.cs" />
     <Compile Remove="libraries\System.IO.FileSystem\Perf.RandomAccess.NoBuffering.cs" />
+    <Compile Remove="libraries\System.Formats.Tar\*.cs" />
```

Comment:
this compile remove does not work for net6.0. I think it would be better to just exclude it for everything that is prior to .NET 7:

```xml
  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' &lt; '7.0')">
    <Compile Remove="libraries\System.Formats.Tar\*.cs" />
  </ItemGroup>
```

Reply:
Strange. Do you know why it does not work?

Reply:
And to make sure I understand your request correctly: You want me to remove all the `Compile Remove` entries that I added to the existing ItemGroups, not just this one, and then add the new one you suggested. Correct?

Reply:
> Do you know why it does not work?

I am not 100% sure, but I think that `&lt; '6.0'` means "everything less than 6.0", but not "equal 6.0". 

> Correct?

exactly!

Reply:
I think that was the case. You'll let me know if it wasn't. 🙂